sdk/security/keyvault/azadmin/backup/example_test.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package backup_test
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/security/keyvault/azadmin/backup"
+)
+
+var client *backup.Client
+
+func ExampleNewClient() {
+	vaultURL := "https://<TODO: your vault name>.managedhsm.azure.net/"
+	cred, err := azidentity.NewDefaultAzureCredential(nil)
+	if err != nil {
+		// TODO: handle error
+	}
+
+	client, err := backup.NewClient(vaultURL, cred, nil)
+	if err != nil {
+		// TODO: handle error
+	}
+
+	_ = client
+}
+
+func ExampleClient_BeginFullBackup() {
+	storageParameters := backup.SASTokenParameter{
+		StorageResourceURI: to.Ptr("https://<storage-account>.blob.core.windows.net/<container>"),
+		Token:              to.Ptr("SAS_TOKEN"),
```

Comment:
Consider making fake values more obviously fake, for example `"<your SAS token>"`

Reply:
fixed in 721f245